docs/api/geo_datasets.csv

```diff
@@ -24,3 +24,4 @@ Dataset,Type,Source,License,Size (px),Resolution (m)
 `Open Buildings`_,Geometries,"Maxar, CNES/Airbus","CC-BY-4.0 OR ODbL-1.0",-,-
 `PRISMA`_,Imagery,PRISMA,-,512x512,5--30
 `Sentinel`_,Imagery,Sentinel,"CC-BY-SA-3.0-IGO","10,000x10,000",10
+`South America Soybean`_,Masks,Sentinel-2,-,-,10
```

Comment:
Any idea what the license is for this dataset?

Reply:
The [paper](https://www.nature.com/articles/s41893-021-00729-z#data-availability) and websites [[1](https://glad.umd.edu/projects/commodity-crop-mapping-and-monitoring-south-america)] [[2](https://glad.earthengine.app/view/south-america-soybean#lon=-60;lat=-20;zoom=5;)] did not specify the license. GLAD Landsat Analysis Ready Data (ARD) indicates that the data is Creative Commons Attribution License (CC BY) [[source](https://www.glad.umd.edu/ard/license-and-disclaimer-not)].

Reply:
What's the connection between SA Soybean and GLAD?

Reply:
In the Code Availability section of the paper:

> Satellite-based soybean classification was carried out using the GLAD Landsat Analysis Ready Data and Tools available at https://glad.geog.umd.edu/ard/home.

Reply:
Ah, but that doesn't tell me the license of the derived product they created